src/assets/sass/objects/_tool.scss

```diff
@@ -139,5 +139,27 @@
 			}
 		}
 	}
+
+	&.tool--selectable{
+		display:block;
+		position:relative;
+
+		textarea {
```

Comment:
According to my first comment, the selector should become something like `#text--selectable--tmp`